crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -232,6 +232,37 @@ impl Schedule {
         self
     }
 
+    /// Suppress warnings and errors that would result from systems in these sets having ambiguities
+    /// (conflicting access but indeterminate order) with systems in `set`.
+    #[track_caller]
+    pub fn ignore_ambiguities<M1, M2, S1, S2>(&mut self, a: S1, b: S2) -> &mut Self
```

Comment:
can we make this just sugar for `configure_sets(a.ambiguous_with(b))`?

Reply:
I wonder if we're being too strict here and should just deny using system type sets with `in_set`. @maniwani any opinion?

Reply:
🤔 I think that's a pretty reasonable way to increase the flexibility of the API. `in_set` is definitely a mistake, but being able to configure public systems without an explicit label seems safe.

Reply:
I don't understand what's being asked here. `in_set` should already fail if given a system type set.

Reply:
Should `configure_set(my_system.before(ArbitraryLabel))` work?

Reply:
Last I remember, if you call `configure_set` or `in_set` with a system type set, it should immediately error at runtime. Maybe somebody changed that, but the error was intentional.

It'd be too easy to call that with a system that's in many places like `apply_deferred`.

For similar reasons, the ambiguity-related methods will result in a build error when there are multiple instances of the system in the schedule.

Reply:
The error on `in_set` should be obvious. Users shouldn't be adding any other kind of system to the type set.

The immediate error on `configure_set` is directed at stopping users from calling `before`, `after`, `run_if`, and `in_set` and it affects systems every instance of a system.

It'd be a little more complicated, but someone could tweak that so that `configure_set` doesn't error if you only call ambiguity methods. I don't if that's generally "safe" to use tho.

Reply:
Sounds good, let's leave it as is for now.